LICENCE

```diff
@@ -0,0 +1,8 @@
+/*******************************************************
+ * Copyright (C) 2017-2018 James Thorogood <jamesthorogood@ewb.ca>
+ *
+ * This file is part of AgriTrader.
+ *
+ * AgriTrader can not be copied and/or distributed without the express
+ * permission of James Thorogood.
+ *******************************************************/
```

Comment:
# Requested change

Please leave a newline at eof

Reply:
Added a newline, as well as a comment about how all files in AgriTrader's directories and subdirectories also belong to AgriTrader.